packages/@lwc/jest-resolver/src/index.js

```diff
@@ -40,26 +40,21 @@ function isImplicitHTMLImport(importee, { basedir }) {
 function isValidCSSImport(importee, { basedir }) {
     const ext = extname(importee);
     const isCSS = ext === '.css';
-    let fileName = basename(importee, '.css');
-    const isScoped = extname(fileName) === '.scoped';
-    if (isScoped) {
-        fileName = basename(fileName, '.scoped');
-    }
     const absPath = resolve(basedir, importee);
-    const dir = dirname(absPath);
-    const jsFile = join(dir, fileName + '.js');
-    const tsFile = join(dir, fileName + '.ts');
 
     return (
         // if it is a css file
         isCSS &&
         // the css file must exist
-        fs.existsSync(absPath) &&
-        // there must be a js/ts file with the same name in the same folder
-        (fs.existsSync(jsFile) || fs.existsSync(tsFile))
+        fs.existsSync(absPath)
     );
 }
 
+function isValidImport(importee, { basedir }) {
```

Comment:
nit: to distinguish from `isValidCSSImport`?
```suggestion
function isValidScriptImport(importee, { basedir }) {
```